### PR TITLE
Update ruleset: BodyBuilding.com

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/BodyBuilding.com.xml [hunk 1]
```diff
@@ -1,73 +1,38 @@
...
+
+	<!--
+		*.bodybuilding.com
+
+		Invalid certificate:
+			- cschat.bodybuilding.com
+			- n.bodybuilding.com: equivalent to bodybuilding-com.112.2o7.net
+	-->
+	<target host="bodybuilding.com" />
 	<target host="*.bodybuilding.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/BodyBuilding.com.xml]: Is there a specific reason for a wildcard here?

Comment (injust) [reply src/chrome/content/rules/BodyBuilding.com.xml]: Not that I can remember, going to enumerate subdomains now.

Comment (injust) [reply src/chrome/content/rules/BodyBuilding.com.xml]: Fixed.

File: src/chrome/content/rules/BodyBuilding.com.xml [hunk 2]
```diff
@@ -1,73 +1,79 @@
...
+	<target host="marketing.bodybuilding.com" />
+	<target host="md.bodybuilding.com" />
+	<target host="mo.bodybuilding.com" />
+	<target host="my.bodybuilding.com" />
+	<target host="n.bodybuilding.com" />
+	<target host="ng.bodybuilding.com" />
+	<target host="om.bodybuilding.com" />
+	<target host="reviews.bodybuilding.com" />
+	<target host="s.bodybuilding.com" />
+	<target host="tr.bodybuilding.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/BodyBuilding.com.xml]:
```
uk.bodybuilding.com
jp.bodybuilding.com

Comment (injust) [reply src/chrome/content/rules/BodyBuilding.com.xml]: Added, thanks!

**Pull request merged**